src/icons/style.css

```
@@ -1,15 +1,18 @@
 @font-face {
-  font-family: "icomoon";
-  src: url("fonts/icomoon.woff?edoxd9") format("woff");
+  font-family: 'icomoon';
+  src:
+    url('fonts/icomoon.woff2?18z8tb') format('woff2'),
+    url('fonts/icomoon.ttf?18z8tb') format('truetype'),
+    url('fonts/icomoon.woff?18z8tb') format('woff'),
+    url('fonts/icomoon.svg?18z8tb#icomoon') format('svg');
```

Thanks to a bug with nextjs static file serving, we inline the font as base64 when building the NDS dist. To keep the size of this file reasonable, we serve only the `woff`.

```suggestion
  src: url('fonts/icomoon.woff?18z8tb') format('woff');
```

---

just updated the CSS!